src/main.rs

```diff
@@ -200,7 +190,10 @@ fn main() {
                 winit::event::WindowEvent::CloseRequested => {
                     *control_flow = ControlFlow::Exit;
                 }
-                _ => {}
+                event => {
+                    // Pass the winit events to the platform integration.
+                    state.on_event(&context, &event);
+                }
```

Comment:
The `State::on_event` function is documented as flagging whether it wan't ownership of the event it is passed, so this should probably be placed before any other event handling.